### PR TITLE
Feat/Survey connector

### PR DESCRIPTION
**Pull request by MelisandeDes**
Add the survey connector to get the comments from the api

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -299,7 +302,9 @@ def populate(ctx, skip_versions,
             # if we use code maat git.setup_aliases(configuration.author_alias)
     
     git.populate_db(skip_versions)
-
+    survey = SurveyAPIConnector(configuration, session)
+    comments = survey.get_comments()
+    survey.save_comments_to_db(comments)
```
Comment (Creadeyh) [inline main.py:307]: Make it one method to call from the outside, like survey.populate_comments()

File: models/comment.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import Column, Integer, String, ForeignKey
+from models.database import Base
+
+
+class Comment(Base):
+    """
+    Comment
+    """
+    __tablename__ = "comments"
```
Comment (Creadeyh) [inline models/comment.py:9]: SQL table names should be singular
```suggestion
    __tablename__ = "comment"
```

File: models/comment.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    comment_id = Column(Integer, primary_key=True)
+    project_name = Column(Integer, ForeignKey("project.name"))
```
Comment (Creadeyh) [inline models/comment.py:11]: What if Survey's project name doesn't match BugPrediction's ?

File: connectors/survey.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import logging
+import requests
+
+from models.comment import Comment
+
+class SurveyAPIConnector:
```
Comment (Creadeyh) [inline connectors/survey.py:6]: Maybe rename SurveyConnector ? Just to keep consistency with the other connectors which also use APIs (git, Jira, etc)

File: .env-example [hunk 5]
```diff
@@ -63,3 +63,8 @@ OTTM_COMMIT_BAD_MSG=fix,ok,commit,test,change,reorg,clean,doc,refresh
 OTTM_LEGACY_PERCENT=20
 # The number of seconds to wait after a failed API call due to a limit of calls exceeded
 OTTM_RETRY_DELAY=3600
+
+# The url for survey api
+SURVEY_BACK_API_URL="http://localhost:8000/"
+# The name of the project for Survey API
+SURVEY_PROJECT_NAME=
```
Comment (Creadeyh) [inline .env-example:70]: Make it possible to give multiple project names

File: README.md [hunk 6]
```diff
@@ -104,6 +104,14 @@ One of the features of the report is to assess the risk of releasing the next ve
 
 See the [list of commands](./docs/commands.md) for other options.
 
+## Connecting with Survey
+
+BugPrediction can be connected with Survey to retrieve user feedback and add it to the database. To do so, you need to start the Survey server and add the following environment variables to the .env file:
```
Comment (Creadeyh) [inline README.md:109]:
```suggestion
BugPrediction can be connected with our [Survey tool](https://github.com/optittm/survey-back-api) to retrieve user feedback and add it to the database. To do so, you need to start the Survey API server and add the following environment variables to the .env file:  
```

**Comment by Creadeyh** [comment]
We should also discuss whether the Comment table should be able to support multiple sources of comments.

**Comment by Creadeyh** [comment]
I merged #46 , you can also add the migration for the comment table here

**Review by FredericDymko — APPROVED** [review]

**Pull request merged**